addon/components/pie-chart.js

```diff
@@ -83,7 +88,7 @@ const PieChartComponent = ChartComponent.extend(FloatingTooltipMixin,
         color: d.color,
         label: d.label,
         value: d.value,
-        percent: d3.round(100.0 * d.value / total, this.get('maxDecimalPlace'))
+        percent: 100.0*d.value / total
```

Comment:
I'm pretty sure D3 has a built in limit, but I think that the developer who is integrating Ember Charts into their system should be defining the maximum number of decimal points allowable from their side.

FYI - Rounding will not increase the number of decimal points if the percentage is precise already. For example, rounding 0.3 to 5 decimal places will still return 0.3 instead of 0.30000 - additional precision will not be provided unless it is necessary